include/xplat/WinObjCRTExport.h

```diff
@@ -15,6 +15,9 @@
 //******************************************************************************
 #pragma once
 
+// OSX has import/export on by default, making these unnecessary
```

Comment:
Can you explain more what this means? I'm sure I understand how this is "on by default" and how we don't at least need to be extern "C" ing things